modules/microshift-configuring-ovn.adoc

```diff
@@ -47,46 +43,21 @@ mtu: 1400
 |Description
 |Example
 
-|`ovsInit.disableOVSInit`
-|bool
-|false
-|Skip configuring OVS bridge `br-ex` in `microshift-ovs-init.service`
-|true ^[1]^
-
-|`ovsInit.gatewayInterface`
-|Alpha
-|eth0
-|Ingress that is the API gateway
-|eth0
-
 |mtu
 |uint32
 |auto
 |MTU value used for the pods
 |1300
 |===
 +
-[.small]
---
-1. The OVS bridge is required. When `disableOVSInit` is true, OVS bridge `br-ex` must be configured manually.
-+
 [IMPORTANT]
 ====
 If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the host that {product-title} is running on to apply the updated setting.
 ====
---
 
 .Example custom `ovn.yaml` configuration file
```

Comment:
Maybe tuck this example under the note?

Reply:
@mburke5678 I don't understand this merge review. You want me to remove the line space? Can you link me to the repo requirement for that, please? I am unfamiliar with the guidance. My understanding is that a line space is needed after the closing of the note? In any case, I hooked these piece together, if that's what you intended?